app/templates/components/forms/admin/settings/ticket-fees-form.hbs

```diff
@@ -0,0 +1,41 @@
+<form class="ui form" {{action 'submit' on='submit'}}>
+  <h3 class="ui header">
+    {{t 'List of supported currencies'}}
+  </h3>
+  {{#each model as |ticketFee|}}
+    <div class="ui sub header">
+      {{get (find-by 'code' ticketFee.currency currencies) 'name'}} - {{ticketFee.currency}} ({{get (find-by 'code' ticketFee.currency currencies) 'symbol'}})
```

Comment:
Can you make two helpers for this ? 
`{{currency-name 'USD'}}`
`{{currency-symbol 'USD'}}`